frontend/src/app/modules/scripts/scripts-page/scripts-page.component.ts

```diff
@@ -140,21 +154,25 @@ export class ScriptsPageComponent extends BaseComponent implements OnInit {
                 takeUntil(this.unsubscribe$),
                 finalize(() => this.spinner.hide()),
             )
-            .subscribe(
-                (updatedScript: ScriptContentDto) => {
-                    this.updateScriptContent(updatedScript.content);
-                    // Clear the error for the current script
+            .subscribe({
+                next: (updatedContent: ScriptContentDto) => {
+                    this.selectedScript!.content = updatedContent.content;
+                    this.scripts[this.scripts.findIndex((s) => s.id === this.selectedScript!.id)].content =
+                        updatedContent.content;
+                    this.form.patchValue({
+                        scriptContent: updatedContent.content,
+                    });
+                    this.form.markAsDirty();
                     if (this.selectedScript) {
                         delete this.scriptErrors[this.selectedScript.id];
                     }
                     this.notification.info('Script content successfully formatted');
                 },
-                (err: ScriptErrorDto) => {
-                    this.notification.error('Format script error');
+                error: (err: ScriptErrorDto) => {
                     this.updateScriptContentError(err);
-                    this.scrollToResult('.script-error');
+                    this.scrollToResult(false);
```

Comment:
if the error is present, probably we should scroll. not sure here

Reply:
it scrolls to the error, but doesn't show error notification. Vlad suggests one error message is enough